EDAVSurveyAndAnalysis.Rmd

```diff
@@ -0,0 +1,29 @@
+# EDAV Survey and Analysis
+Varalika Mahajan and Vrinda Bhat
```

Comment:
Per the instructions, The second line should be blank, followed by the author name(s).